main.go

```diff
@@ -46,22 +50,37 @@ var RootCmd = &cobra.Command{
 	Example: "heketi --config=/config/file/path/",
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Printf("Heketi %v\n", HEKETI_VERSION)
-		if !showVersion {
+		if showVersion {
+			// Quit here if all we needed to do was show version
+			os.Exit(0)
+		} else {
 			// Check configuration file was given
-			if configfile == "" {
+			if configfile == "" && editMode == false {
 				fmt.Fprintln(os.Stderr, "Please provide configuration file")
 				os.Exit(1)
 			}
-		} else {
-			// Quit here if all we needed to do was show version
-			os.Exit(0)
+			// Check args for edit mode
+			if editMode == true {
+				fmt.Fprintf(os.Stderr, "rtalur testing")
+				if jsonFile == "" {
+					fmt.Fprintln(os.Stderr, "Please provide file for input")
+					os.Exit(1)
+				}
+				if dbFile == "" {
+					fmt.Fprintln(os.Stderr, "Please provide path for db file")
+					os.Exit(1)
+				}
+			}
 		}
 	},
 }
 
 func init() {
 	RootCmd.Flags().StringVar(&configfile, "config", "", "Configuration file")
 	RootCmd.Flags().BoolVarP(&showVersion, "version", "v", false, "Show version")
+	RootCmd.Flags().BoolVarP(&editMode, "editmode", "", false, "Runs heketi in database edit mode")
```

Comment:
If I may bikeshed about this option name for a moment: I think it would be nice to have the option name refer to the fact that its the db being edited. Something like "--dbedit"  or "--load-db" just "--editmode" by itself doesn't make a mental connection with db (at least for me).

Reply:
I think this should be named `--import` or `--import-db`: It's not an editmode of heketi, but heketi importing a (possibly edited) json dump of the db.

Reply:
done.